docs/Authentication.md

```diff
@@ -23,18 +21,17 @@ which is usually not necessary. This might be a better idea:
 
 ```swift
 let provider = MoyaProvider<YourAPI>(plugins: [CredentialsPlugin { target -> URLCredential? in
-    switch target {
-      case .targetThatNeedsAuthentication:
-        return URLCredential(user: "user", password: "passwd", persistence: .none)
-      default:
-        return nil
+        switch target {
```

Comment:
Wasn't sure here - as it is a closure, with the `])` on the last line, I thought this would be better than two `}`'s on the same space vertically. 🤔